### PR TITLE
feat: add rating component

### PR DESCRIPTION
**Pull request by RakshaPawar108**
Added Rating component. Would love feedback and suggestions!
Screenshot (109)

Screenshot (110)

### CODE REVIEW

**Review by mohammed-sas — APPROVED** [review]
good to go

**Review by Abhijeet43 — APPROVED** [review]
Looks fine!!

File: components/Rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.ratings-wrapper {
+	align-items: center;
+	align-self: center;
+	background-color: var(--white);
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	justify-self: center;
```
Comment (Ayushi20-19) [inline components/Rating/rating.css]: I thing this property is not in use here as u are giving it in wrapper itself, and justify content already making it center! please correct me if i m wrong.

**Review by Ayushi20-19 — APPROVED** [review]

**Review by VishalPatil18 — APPROVED** [review]
The code looks clean and also works great. Well done @RakshaPawar108 👍

File: components/Rating/rating.css (hunk 1 above)
Comment (RakshaPawar108) [reply components/Rating/rating.css]: Ohh yes! that's right. My bad. I'll remove it. Thanks for the review.

**Comment by RakshaPawar108** [comment]
> The code looks clean and also works great. Well done @RakshaPawar108 👍

Thanks Vishal

**Pull request merged**